src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/TextField/BitTextFieldDemo.razor.cs

```diff
@@ -482,6 +491,17 @@ public class ValidationTextFieldModel
     public string RangeText { get; set; }
 }";
 
+        private readonly string example14HTMLCode = @"<div class=""example-desc"">The trim property removes whitespace from both sides of a string.</div>
```

Comment:
Plz modify the desc:
`The trim property removes whitespaces from both sides of a string.`